src/components/modals/HarvestAll.tsx

```diff
@@ -407,19 +407,28 @@ export const HarvestAllModal: FC<IHarvestAllModalProps> = ({
 											{tokenSymbol}
 										</BreakdownUnit>
 										<BreakdownRate>
-											{formatWeiHelper(
-												rewardStream,
-												config.TOKEN_PRECISION,
-												false,
-											)}
+											{Number(
+												formatWeiHelper(
+													rewardStream,
+													config.TOKEN_PRECISION,
+													false,
+												),
+											) -
+												Number(
+													formatWeiHelper(
+														givBackStream,
+														config.TOKEN_PRECISION,
+														false,
+													),
+												)}
```

Comment:
Thank you @divine-comedian 
I suggest this solution:
```
{formatWeiHelper(
  rewardStream.minus(
	  givBackStream,
  ),
  config.TOKEN_PRECISION,
  false,
)}
```

Reply:
thank you, made the changes!